v4/newrelic/application.go

```diff
@@ -120,5 +145,42 @@ func NewApplication(opts ...ConfigOption) (*Application, error) {
 			propagation.WithInjectors(trace.TraceContext{}),
 			propagation.WithExtractors(trace.TraceContext{}))
 	}
-	return &Application{tracer: tracer, propagators: propagators}, nil
+
+	if c.Logger == nil {
+		c.Logger = logger.ShimLogger{}
+	}
+	msg := fmt.Sprintf("Starting New Relic Shim Agent. If everything is configured properly, you should be able"+
+		" to see your data in a few minutes by visiting https://one.newrelic.com/launcher/nr1-core.explorer and"+
+		" searching for '%s'.", c.AppName)
+	c.Logger.Info(msg, nil)
+
+	return &Application{
+		tracer:      tracer,
+		propagators: propagators,
+		logger:      c.Logger,
+	}, nil
+}
+
+func (app *Application) logInfo(msg string, context map[string]interface{}) {
+	if app != nil {
+		app.initLogger()
+		app.logger.Info(msg, context)
+	}
+}
+
+func (app *Application) logDebug(msg string, context map[string]interface{}) {
```

Comment:
This is now not exported. Did you decide against having these logging methods exposed to users of the API? I'd be interested in what you found that led to the reversal. (There's a definite advantage to not exporting them assuming we are confident that their absence will not be blocking to any users of the agent).

Reply:
Yep, I took a look at the v3 agent and realized that we don't expose any logging to the customer. If they want to log something, they can do it in their own way; no need to expose a way for them to log specifically via the agent. I don't remember why I thought it should be exported to start with, but it does not appear to be necessary.